lib/doc/worksheet.js

```diff
@@ -613,20 +613,23 @@ class Worksheet {
   }
 
   _mergeCellsInternal(dimensions, ignoreStyle) {
-    // check cells aren't already merged
-    _.each(this._merges, merge => {
-      if (merge.intersects(dimensions)) {
-        throw new Error('Cannot merge already merged cells');
-      }
-    });
-
     // apply merge
     const master = this.getCell(dimensions.top, dimensions.left);
+    // check cells aren't already merged
+    if (master.isMerged()) {
```

Comment:
isMerged is not a function, its  a getter so you need this.

```suggestion
    if (master.isMerged) {
```

Reply:
@markygab I'm sorry, I didn't notice your review until now, and I am super busy at work and home with something else. Are you or anyone else able to apply your suggestions and test them? I recall that I did test my functionality in my own application, and it seemed to work, but perhaps I just didn't notice there was a bug somewhere.

Reply:
Or, if you need me to, I could use the github UI to apply suggestions, so you could review again. Let me know. :)